Example/directManipulation/shuffle.js

```diff
@@ -0,0 +1,307 @@
+import React, { useState, useRef, useMemo, useEffect, useCallback } from 'react';
+import { StyleSheet, Button, findNodeHandle, UIManager, processColor, Platform, StatusBar } from 'react-native';
+import Animated, { Transitioning, Transition, TransitionState } from 'react-native-reanimated';
```

Comment:
So AFAIK this example is no longer relevant right?

Reply:
Very relevant. I've changed it to re-evaluate with a timeout instead of once the transition event has been fired.

Reply:
See the line in the next review